fastlane/Fastfile

```diff
@@ -74,10 +83,43 @@ end
 
 platform :android do
   lane :code_freeze do | options |
-    android_codefreeze_prechecks(options)
+    ensure_git_status_clean
+
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    confirmation_message = <<-MESSAGE
+
+      Code Freeze:
+      • New release branch from #{DEFAULT_BRANCH}: release/#{next_release_version}
+      • Current release version and build code: #{current_release_version} (#{current_build_code}).
+      • New release version and build code: #{first_beta_version} (#{next_build_code}).
+
+      Do you want to continue?
+
+    MESSAGE
+
+    # Ask user confirmation
+    UI.user_error!('Aborted by user request') unless options[:skip_confirm] || UI.confirm(confirmation_message)
+
+    # Create the release branch
+    UI.message 'Creating release branch...'
+    ensure_git_branch(branch: DEFAULT_BRANCH)
+    Fastlane::Helper::GitHelper.create_branch("release/#{next_release_version}", from: DEFAULT_BRANCH)
+    UI.message "Done! New release branch is: #{git_branch}"
+
+    # Bump the version and build code
+    UI.message 'Bumping beta version and build code...'
+    Fastlane::Helper::GitHelper.ensure_on_branch!('release')
```

Comment:
Looks like we use `ensure_git_branch` above, but this here – can we make that consistent?

Reply:
Addressed in https://github.com/Automattic/pocket-casts-android/pull/1425/commits/29bb7e4e334991f5b6cca3fa550fb74905f0cec2